src/DependencyInjection/Configuration.php

```diff
@@ -46,6 +46,8 @@ public function getConfigTreeBuilder()
                             ->scalarNode('proxy')->defaultValue('')->end()
                             ->booleanNode('http_errors')->defaultValue(true)->end()
                             ->floatNode('timeout')->defaultValue(5.0)->end()
+                            ->floatNode('connect_timeout')->defaultValue(30.0)->end()
```

Comment:
it feels weird to set a `connect_timeout` value to `30` whereas the `timeout` default value is only `5`

something like `1.0` or `2.0` as default should be long enough, no?

Reply:
I set it to 5.0 both as Benoit suggested me